Dockerfile

```diff
@@ -28,11 +28,33 @@ FROM base as development
 
 CMD [ "poetry", "run", "flask", "run", "--host=0.0.0.0", "--port=5000", "--debugger"] 
 
+# testing stage
+FROM base as test
+RUN pip install pytest
+RUN poetry add pytest --dev
+#WORKDIR /tests
+ENV GECKODRIVER_VER v0.31.0
+# Install the long-term support version of Firefox (and curl if you don't have it already)
+RUN apt-get update && apt-get install -y firefox-esr curl
+  
+# Download geckodriver and put it in the usr/bin folder
+RUN curl -sSLO https://github.com/mozilla/geckodriver/releases/download/${GECKODRIVER_VER}/geckodriver-${GECKODRIVER_VER}-linux64.tar.gz \
+   && tar zxf geckodriver-*.tar.gz \
+   && mv geckodriver /usr/bin/ \
+   && rm geckodriver-*.tar.gz
+RUN  pip install selenium   
```

Comment:
(Until we've actually got some running e2e tests, it could be worth getting rid of this which will cut a chunk of time off your image builds, but it's pretty minor so definitely optional!)